test/hotspot/gtest/utilities/test_globalDefinitions.cpp

```diff
@@ -304,8 +304,8 @@ TEST(globalDefinitions, jlong_from) {
   EXPECT_EQ(val, 0x00000000000000FF);
 
   val = jlong_from(0xFFFFFFFF, 0);
-  EXPECT_EQ(               val, (signed long)0xFFFFFFFF00000000);
-  EXPECT_EQ((unsigned long)val,              0xFFFFFFFF00000000);
+  EXPECT_EQ(                    val, (signed long long)0xFFFFFFFF00000000);
```

Comment:
Oops! Sorry I missed that these tests were previously casting to `long`
variants. Never use `long` in shared code. But we also almost never use `long
long` in shared code. (And a couple of the uses look a bit questionable. I'll
be following up on those.) Use CONST64 and UCONST64 to form the 64 bit
constants, rather than casts.